chart/templates/drupal-configmap.yaml

```diff
@@ -13,171 +13,53 @@ data:
-    max_file_uploads = 20
-    allow_url_fopen = On
-    allow_url_include = Off
-    default_socket_timeout = 60
-
-    [CLI Server]
-    cli_server.color = On
 
     [Date]
     date.timezone = Europe/Helsinki
 
-    [Pdo_mysql]
-    pdo_mysql.cache_size = 2000
-    pdo_mysql.default_socket =
-
     [mail function]
     mail.add_x_header = Off
 
-    [SQL]
-    sql.safe_mode = Off
-
-    [MySQLi]
-    mysqli.max_persistent = -1
-    mysqli.allow_persistent = On
-    mysqli.max_links = -1
-    mysqli.cache_size = 2000
-    mysqli.default_port = 3306
-    mysqli.reconnect = Off
-
-    [mysqlnd]
-    mysqlnd.collect_statistics = On
-    mysqlnd.collect_memory_statistics = Off
-
-    [PostgreSQL]
-    pgsql.allow_persistent = On
-    pgsql.auto_reset_persistent = Off
-    pgsql.max_persistent = -1
-    pgsql.max_links = -1
-    pgsql.ignore_notice = 0
-    pgsql.log_notice = 0
-
-    [bcmath]
-    bcmath.scale = 0
-
     [Session]
-    session.save_handler = files
-    session.save_path = /tmp
-    session.use_strict_mode = 0
-    session.use_cookies = 1
-    session.use_only_cookies = 1
-    session.name = PHPSESSID
-    session.auto_start = 0
     session.cookie_lifetime = 0
-    session.cookie_path = /
-    session.cookie_domain =
     session.cookie_httponly = 1
     session.serialize_handler = php_binary
-    session.gc_probability = 1
     session.gc_divisor = 10000
     session.gc_maxlifetime = 1440
-    session.referer_check =
-    session.cache_limiter = nocache
-    session.cache_expire = 180
-    session.use_trans_sid = 0
-    session.hash_function = 0
-    session.hash_bits_per_character = 5
-    url_rewriter.tags = "a=href,area=href,frame=src,input=src,form=fakeentry"
 
     [Assertion]
     zend.assertions = -1
 
     [mbstring]
     mbstring.internal_encoding = UTF-8
 
-    [Tidy]
-    tidy.clean_output = Off
-
-    [soap]
-    soap.wsdl_cache_enabled = 1
-    soap.wsdl_cache_dir = "/tmp"
-    soap.wsdl_cache_ttl = 86400
-    soap.wsdl_cache_limit = 5
-
-    [ldap]
-    ldap.max_links = -1
-
     [opcache]
     opcache.enable = 1
     opcache.enable_cli = 0
     opcache.memory_consumption = 128
     opcache.interned_strings_buffer = 32
-
     opcache.use_cwd = ${PHP_OPCACHE_USE_CWD}
     opcache.validate_timestamps = 0
-    opcache.enable_file_override = 1
     opcache.log_verbosity_level = 2
 
     [igbinary]
     igbinary.compact_strings=1
 
-
-  php-fpm_conf: |
-    [global]
-    error_log = /proc/self/fd/2
-    log_level = {{ .Values.php.php_ini.loglevel }} ; Possible Values: alert, error, warning, notice, debug
-    daemonize = no
-    include=/etc/php7/php-fpm.d/*.conf
-
-  www_conf: |
-    [www]
-    listen = [::]:9000
-    user = www-data
-    group = www-data
-    pm = ondemand
-    pm.max_children = 100
-    pm.start_servers = 5
-    pm.min_spare_servers = 5
-    pm.max_spare_servers = 20
-    pm.process_idle_timeout = 60s
-    pm.max_requests = 500
-    access.log = /proc/self/fd/2
```

Comment:
We need this override

Reply:
These overrides were not used at all for a while. The default values are here: https://github.com/amazeeio/lagoon/blob/db05a8cad8cdb0f36e2d16a462a7ef83e1450f3f/images/php/fpm/php-fpm.d/www.conf, and before we override them we should have a clear reasoning for doing it.